### ISSUE TITLE
Documentation modules

### ISSUE THREAD

**Issue by davide97g** [davide97g/ml-es6-modules#4]
# Documentation

Modules are working and ready to use. 
It would be _sad_ if the only one able to use them was me.
Documentation is **key**: just describe the usage process and briefly how the methods work.

## A good doc
The final user should know, in descending order of importance :
- **README** module:
  - [ ] module information
  - [ ] usage tutorial
  - [ ] developer instruction
- Inside file with **jsdoc**:
  - [ ] classes/functions description
  - [ ] methods description

To sum up things:
- [ ] a clean and simple README of all the modules

would be perfect: give an introduction without becoming module-specific.
